src/mapReducers.js

```diff
@@ -1,4 +1,5 @@
 import _ from 'lodash';
+import { isTargetAllAction } from './context';
```

Comment:
We must have reserved property for such action because we want to target all reducers with single action, and it is not possible to set keySelector values to proper value for all reducers with single action (we do not know what keySelectors exists).
There is also a case when you want to target a map reducers group, then value of keySelector could be checked to target all mapped reducers, but it is different then what am I solving with this PR.